src/helpers/fixtures.js

```diff
@@ -60,7 +60,13 @@ let fixtureHelper = module.exports = {
         }
       }
 
-      if (options.query && !_.isEqual(options.query, fixture.request.query)) {
+      const filteredFixturedQuery = xhrHelper.getQueryStringObject({
+        url: 'http://localhost/?' + Object.keys(fixture.request.query || {}).map(function(key) {
+          return key + '=' + fixture.request.query[key];
+        }).join('&')
```

Comment:
Is it worth pulling in [qs](https://www.npmjs.com/package/qs) for this task? I'm not too fussed about asset size on this library as it's not intended for production use.